### PR TITLE
Use alert-link class, change link color for default warning alert

### PR DESCRIPTION
**Pull request by mbabker**
#### Summary of Changes

Consistently use recently introduced `alert-link` class for all links in rendered alert divs.  Also, changes the color of alert links for the default warning alert div to match its color (and Bootstrap 3's implementation)

Before:
screen shot 2016-07-19 at 3 25 17 pm

After:
screen shot 2016-07-19 at 3 25 00 pm
#### Testing Instructions

In the update component, the link to the download page should now have the "bold" appearance.  Same for the link in the stats plugin's nag message and duplicate queries in the debug console's output.  Links in warning alerts (default yellow) should now have the brownish appearance instead of blue.

As for the stray CSS change with the `.categories-list .collapse` selector, that came after running the `generatecss.php` script.  No idea why else it'd be there.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_joomlaupdate.ini [hunk 1]
```diff
@@ -55,7 +55,7 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_UPDATES_INFO_CUSTOM="You are on the &quot;%s&quot;
...
 COM_JOOMLAUPDATE_VIEW_DEFAULT_UPDATES_INFO_TESTING="You are on the &quot;%s&quot; update channel. This channel is designed for testing new releases and fixes in Joomla.<br />It is only intended for JBS (Joomla Bug Squad&trade;) members and others within the Joomla community who are testing. Do not use this setting on a production site."
-COM_JOOMLAUPDATE_VIEW_DEFAULT_UPLOAD_INTRO="You can use this feature to update Joomla if your server is behind a firewall or otherwise unable to contact the update servers. First download the Joomla <em>Upgrade Package</em> in ZIP format from <a href=\"%s\">the official Joomla download page</a>. Then use the fields below to upload and install it."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_UPLOAD_INTRO="You can use this feature to update Joomla if your server is behind a firewall or otherwise unable to contact the update servers. First download the Joomla <em>Upgrade Package</em> in ZIP format from <a class=\"alert-link\" href=\"%s\">the official Joomla download page</a>. Then use the fields below to upload and install it."
```
Comment (dgt41) [inline administrator/language/en-GB/en-GB.com_joomlaupdate.ini]: `\"` 💯
So someone needs to do a huge find and replace 😄

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 0012d075684f7bd1f209f679d2bf8bccf4003e4b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Comment by RonakParmar** [comment]
I have tested for _"Same for the link in the stats plugin's nag message and duplicate queries in the debug console's output."_. 
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Comment by killoltailored** [comment]
I have tested this item :white_check_mark: successfully on 0012d075684f7bd1f209f679d2bf8bccf4003e4b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Comment by killoltailored** [comment]
Tested successfully for _"stats plugin's nag message and duplicate queries in the debug console's output"_  <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Comment by RonakParmar** [comment]
Where I can see the above warning message which you have displayed in screen-dump in description?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Comment by zero-24** [comment]
@RonakParmar here: https://issues.joomla.org/tracker/joomla-cms/add

**Comment by RonakParmar** [comment]
@zero-24  Thank you for the Link, but How to test this link after applying the PR? I have replicated latest staging site in my local system and using patch tester component I am testing all PRs.
So, how can I test this warning message in my local? Or what should I have to do to test the warning message?
Thank you.

**Comment by zero-24** [comment]
@RonakParmar this screenshot was just an example how it looks like ;) for testing you can have a look to the `Testing Instructions` ;)

**Comment by RonakParmar** [comment]
Okay got it, Apologize for the misunderstanding, Then I have already tested using system debug console. Thanks for the clarification @zero-24  :+1:

**Comment by truptikagathara** [comment]
I have tested this item :white_check_mark: successfully on 0012d075684f7bd1f209f679d2bf8bccf4003e4b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11209">issues.joomla.org/joomla-cms/11209</a>.</sub>

**Pull request merged**